src/context.rs

```diff
@@ -1380,7 +1380,7 @@ impl Context {
             random.truncate(buffer.size.try_into().unwrap()); // should not panic given the TryInto above
             Ok(Digest::try_from(random)?)
         } else {
-            error!("Error in flushing context: {}.", ret);
+            error!("Error in getting random bytes: {}.", ret);
```

Comment:
The error message was random

![image](https://user-images.githubusercontent.com/29229160/101478382-8236f380-3948-11eb-9ac7-0b0508628a2d.png)

Reply:
Indeed, I kept trying to understand where a context was being flushed 😅